client/blocks/taxonomy-manager/style.scss

```diff
@@ -36,63 +36,70 @@
 
 .taxonomy-manager__list-item-card {
 	cursor: pointer;
+	padding: 0 !important;
```

Comment:
Why `!important`? I'm guessing you had a [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) issue, in which case you should try making your selector more specific (perhaps by prepending the selector you need to override). We should try to avoid `!important` as much as possible, because inevitably you need something that is important-_er_